OpenAI/Packages/com.openai.unity/Runtime/Assistants/AssistantsEndpoint.cs

```diff
@@ -38,7 +38,11 @@ public async Task<ListResponse<AssistantResponse>> ListAssistantsAsync(ListQuery
         public async Task<AssistantResponse> CreateAssistantAsync(CreateAssistantRequest request = null, CancellationToken cancellationToken = default)
         {
             request ??= new CreateAssistantRequest();
+            ResponseFormatObject responseFormatObject = new ResponseFormatObject(ResponseFormat.Json);
+            var json = JsonConvert.SerializeObject(responseFormatObject, OpenAIClient.JsonSerializationOptions);
+            UnityEngine.Debug.Log(json);
             var jsonContent = JsonConvert.SerializeObject(request, OpenAIClient.JsonSerializationOptions);
+            UnityEngine.Debug.Log(jsonContent);
```

Comment:
JFYI These two debug lines can probably be removed in final PR.

Reply:
np, was planning on it. Still working out the last changes needed